examples/index.jsx

```diff
@@ -21,7 +23,10 @@ const initialValues = {
 
 const Main = () => {
   return (
-    <FormApp basename="/" initialValues={ initialValues } />
+    <>
+      <FormApp basename="/" initialValues={ initialValues } />
+      <ChapterForm basename="/" initialValues={ initialValues } />
```

Comment:
So do both of these show up? How does someone get to the FormApp compared to the ChapterForm? Maybe switch the basename for the chapterform to be something unique so you can navigate to either one?

Reply:
@bjmccotter7192 At the moment, the chapter form is showing alongside the formapp at the bottom of the page:
<img width="825" alt="Screen Shot 2022-05-11 at 10 04 19 AM" src="https://user-images.githubusercontent.com/2727868/167906858-a319b7c2-eb11-4882-b1b1-a242ffebb4d8.png">

It seems to me that this would require to have the BrowserRouter router at a higher level in the component tree. If we're going to be embedding these routes inside an existing application, it might be worth it to have the browser router at the highest possible level for demo purposes, but the details of how it will later integrate are still pretty fuzzy.

For the moment I can move it into the multipage form app, or I can leave it out and we can add it later, since it is covered by the tests.

Reply:
Ahhhh i see i didn't realize that it was appending at the bottom of the form. I don't know if we need this under the multi-page form since were using that as our working demo form.

Reply:
cool, I'll go ahead and remove it since the same content and functionality is tested in the chapter test, and we can add chapters later as needed.